docs/index.md

```diff
@@ -14,7 +14,7 @@ keywords:
 
 ## What is Ensono Stacks
 
-Stacks is an Ensono open-source project that automates the setup of software development projects. Stacks is for software engineers of all kinds who want to create projects quickly.
+Ensono Stacks is an Ensono open-source project that automates the setup of software development projects. Ensono Stacks is for software engineers of all kinds who want to create projects quickly.
 
 Setting up the infrastructure and frameworks necessary to deliver a software project takes time. A team might spend the first two weeks (sprint 0) making technical decisions, provisioning and configuring environments, pipelines, compute, test frameworks, and more. Stacks automates this setup using infrastructure-as-code to shrink your project's sprint 0 from two weeks to an hour or less.
```

Comment:
There is a reference to just 'Stacks' on its own in this section, where it says 'Stacks automates'

Reply:
Also a reference to just stacks on line 33.